libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardLayout/utils/sizing.ts

```diff
@@ -203,20 +231,26 @@ export function splitDashboardLayoutItemsAsRenderedGridRows<TWidget>(
 
 /**
  * Calculate dashboard layout item height for the provided screen.
- * Result is width of the item, defined as grid items count,
- * multiplied by height, defined as a ratio.
+ * Result, if custom height is defined, is height of the item, defined
+ * as grid items count multiplied by 20 or width of the item,
```

Comment:
Nitpick: can we avoid the 20 here so that if we change the constant this stays consistent?

Reply:
Replaced with constant name.